crates/bevy_math/src/curve/adaptors.rs

```diff
@@ -18,6 +18,10 @@ mod paths {
     pub(super) const THIS_CRATE: &str = "bevy_math";
 }
 
+// Import `CurveExt` just for doc links.
```

Comment:
Nit: I recommend using `, reason = "" ` instead of a comment.

Reply:
Good point!